src/main/java/edu/kis/powp/jobs2d/command/manager/DriverCommandManager.java

```diff
@@ -78,4 +84,42 @@ public synchronized String getCurrentCommandString() {
 	public Publisher getChangePublisher() {
 		return changePublisher;
 	}
+
+	// Class representation of Command needed to use with Gson
+	class Command {
+		String operation;
+		int X;
+		int Y;
+	}
+
+	/**
+	 *
+	 * @param filePath path where file with json is stored
+	 * @param name name of the command
+	 */
+	public synchronized void loadFromJsonFile(String filePath, String name) {
+		Gson gson = new Gson();
+		Type myType = new TypeToken<Collection<Command>>() {}.getType();
+		String jsonString = null;
+		try {
+			jsonString = new Scanner(new File(filePath)).useDelimiter("\\Z").next(); // 1-liner that reads whole file to string
+		} catch (FileNotFoundException e) {
+			e.printStackTrace();
+		}
+		List<Command> myCommands = gson.fromJson(jsonString, myType);
+		List<DriverCommand> myDriverCommands = new ArrayList<>();
+		for (Command command : myCommands) {
+			switch (command.operation) {
+				case "SetPositionCommand":
+					myDriverCommands.add(new SetPositionCommand(command.X, command.Y));
+					break;
+				case "OperateToCommand"	:
+					myDriverCommands.add(new OperateToCommand(command.X, command.Y));
+					break;
+			}
+		}
```

Comment:
Please add string->command conversion as a separate (independent) function. e.g., extract to a separate class. Consider introducing an interface to open (OCP) for other formats (e.g., xml) in the future.